src/overlaybd/fs/lsmt/file.cpp

```diff
@@ -654,20 +663,20 @@ class LSMTFile : public LSMTReadOnlyFile {
         return LSMTReadOnlyFile::pread(buf, count, offset);
     }
 
-    virtual void append_index(const SegmentMapping &m) {
-
+    virtual void append_index(const SegmentMapping &m){
```

Comment:
please adjuest the format of the method, eg, `{` and blank after `}` for line 679